modules/data.land/NAMESPACE

```diff
@@ -49,12 +49,23 @@ export(soil.units)
 export(soil2netcdf)
 export(soil_params)
 export(soil_process)
+export(soilgrids_soilC_extract)
 export(subset_layer)
 export(to.Tag)
 export(to.TreeCode)
 export(write_ic)
 export(write_veg)
 importFrom(magrittr,"%>%")
 importFrom(ncdf4,ncvar_get)
+importFrom(reshape2,melt)
 importFrom(rlang,"%||%")
 importFrom(rlang,.data)
+importFrom(stats,optim)
+importFrom(stats,qgamma)
+importFrom(terra,extract)
+importFrom(terra,project)
+importFrom(terra,rast)
+importFrom(terra,vect)
+importFrom(utils,flush.console)
+importFrom(utils,setTxtProgressBar)
+importFrom(utils,txtProgressBar)
```

Comment:
PEcAn house style is to call all of these directly from their namespaces, e.g. `stats::optim(...)` rather than using `importFrom`. The exceptions are cases where the `::` notation doesn't work at all, e.g. `%>%` or `.data`.